client/src/components/Comment.vue

```diff
@@ -63,7 +63,7 @@
     <div data-cy="comment-footer">
       <!-- Don't show the reply button if it's a reply -->
       <div
-        v-if="getUserId && comment.userId != getUserId && !comment?.relatedTo"
+        v-if="getUserId && !comment?.relatedTo"
```

Comment:
Hat das noch einen anderen Grund außer syntactic sugar?

Reply:
Siehe Commit Kommentar "show reply button also for original reply owner". In deinem Beispiel konnte der Reply Owner noch selbst auf seinen Kommentar antworten. Das war aber jetzt nicht mehr möglich.